client/src/components/Header/index.js

```diff
@@ -13,56 +13,56 @@ function Header() {
   };
 
   return (
-    <header className='background-color-secondary'>
```

Comment:
@jojobautistaum you are adding double quotes to this code. Please correct this and push a new commit.

Reply:
Not sure what happen to this. I did a git pull origin develop a couple times. Edison and Henry are even watching when I did that again before the PR. My local code uses a double quote. Anyway, I will re-push and see what happen.